src/views/TechnicalAssistance/TaskList.tsx

```diff
@@ -100,18 +101,24 @@ function TaskList() {
                   <TaskListDescription>
                     <p>{taskListText[0].text}</p>
                   </TaskListDescription>
-
-                  <UswdsReactLink
-                    variant="unstyled"
-                    className="usa-button"
-                    to={`/trb/requests/${id}`}
-                  >
-                    {t(
-                      formStatus === 'READY_TO_START'
-                        ? 'button.start'
-                        : 'button.continue'
-                    )}
-                  </UswdsReactLink>
+                  {/* Continue to fill out the request form or view the submitted request if it's completed */}
+                  {formStatus === TRBFormStatus.COMPLETED ? (
+                    <UswdsReactLink to={`/trb/requests/${id}/view`}>
+                      {t('taskList.viewSubmittedTrbRequest')}
+                    </UswdsReactLink>
```

Comment:
This link should open in a new tab?  Not sure if you had a convo with UX on this.  Either way the verbiage needs to be updated or it needs to open in a new tab

Reply:
This has had `target="_blank"` added to it, resolving!